serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/mocking-http-cloudevents-with-wiremock.adoc

```diff
@@ -1 +1,194 @@
+:event_driven_architecture: https://en.wikipedia.org/wiki/Event-driven_architecture
+:kogito_sw_order_processing_example: {kogito_sw_examples_url}/serverless-workflow-order-processing
+:kogito_sw_order_processing_example_VerifyWorkflowExecutionIT_class: {kogito_sw_order_processing_example}/src/test/java/org/kie/kogito/examples/sw/orders/processing/VerifyWorkflowExecutionIT.java
+:kogito_sw_order_processing_example_pom: {kogito_sw_order_processing_example}/pom.xml
+:kogito_sw_order_processing_example_test_application_properties: {kogito_sw_order_processing_example}/src/test/resources/application.properties
+:recap_of_http_based_testing_in_jvm_mode: {quarkus_testing_guide}#recap-of-http-based-testing-in-jvm-mode
+:awaitility: http://www.awaitility.org/
+:awaitility_class: https://javadoc.io/static/org.awaitility/awaitility/4.2.0/org/awaitility/Awaitility.html
+:wiremockserver_class: https://javadoc.io/doc/com.github.tomakehurst/wiremock/latest/com/github/tomakehurst/wiremock/WireMockServer.html
+:wiremock: https://wiremock.org/
+:wiremock_verifying: {wiremock}/docs/verifying/
+
+This document describes how to test your Serverless Workflow application that uses HTTP {cloud_events_url}[*CloudEvents*] and {knative_sink_binding_impl}[*sink binding*].
+
+The testing case followed in this document is based on the {kogito_sw_order_processing_example}[`serverless-workflow-order_processing`] example application.
+
+[[sinkbinding-test-overview]]
+== Overview of HTTP CloudEvents, SinkBinding and testing
+--
+The application we want to test must be configured to use {knative_eventing}[*Knative Eventing*].
+Using standard HTTP POST requests to send and receive events between event producers and {knative_sink_binding_overview}[*sinks*].
+These events conform to the {cloud_events_url}[*CloudEvents*] specifications, which enables creating, parsing, sending, and receiving events in any programming language.
+
+When you create an event source, you can specify a {knative_sink_binding_overview}[*sink*] where events are sent to, from the source.
+A {knative_sink_binding_overview}[*sink*] is an Addressable or a Callable resource that can receive incoming events from other resources.
+Knative Services, Channels, and Brokers are all examples of sinks.
+In this guide we are covering the testing of the Knative Service configured as {knative_sink_binding_overview}[*sink*].
+
+The purpose of this guide is to mock that service to verify the cloud events are correctly received by the sink.
+The {wiremock}[*Wiremok*] framework will add the mocked server that will verify the different Cloud Events received by the sink in the Serverless Workflow service execution.
+--
+
+[[mocking_sink_binding]]
+== Testing the application with {knative_sink_binding_impl}[*SinkBinding*]
+
+.Prerequisites
+* A completed Kogito Serverless Workflow application working. More information available in {getting_started_create_first_workflow_guide}[Creating your first Serverless Workflow service] guide.
```

Comment:
I think it's useful also to link the events guide here (although to be published 😅 )